backend/src/org/commcare/util/CommCareSession.java

```diff
@@ -721,6 +721,21 @@ public boolean isViewCommand(String command) {
         // commcare core for release issues
```

Comment:
This comment makes me sad! Can we figure out if the "".equals check is needed or not and address the comment?